fenix.json.example

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
I like it as it is. It explains it is fenix not just another plain config.json.example and fenix.json would let me guess it is something which is useful for everybody, which isn't